cloud/aws/terraform/cloudtrail/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "aws" {
+  region = var.region_name
+  default_tags {
+    tags = {
+      module      = module.env.module_name
+      provisioner = "terraform"
+      stage       = terraform.workspace
+    }
+  }
+}
+
+module "processor" {
+  source                = "../common/s3proc"
+  region_name           = var.region_name
+  source_name           = "cloudtrail"
+  source_bucket_name    = var.source_bucket_name
+  source_bucket_region  = var.source_bucket_region
+  vast_lambda_name      = var.vast_lambda_name
+  vast_lambda_arn       = var.vast_lambda_arn
+  vast_lambda_role_name = var.vast_lambda_role_name
+  import_cmd            = local.import_cmd
+}
```

Comment:
The cloudtrail module is basically just a common/s3proc module with specific inputs. In Terragrunt we can usually specify this by using the `source` field of the `terraform` block in the `terragrunt.hcl` file. This would avoid us all this boilerplate with the repetition of the variables in the `inputs.tf` file, but the way it is actually implemented in Terragrunt is incompatible with Terraform Cloud and especially the "Terraform Working Directory" that we use there.